tests/aws/services/lambda_/event_source_mapping/test_lambda_integration_kinesis.py

```diff
@@ -1107,15 +1110,26 @@ def test_kinesis_maximum_record_age_exceeded(
             "StreamDescription"
         ]["StreamARN"]
 
+        if not is_aws_cloud():
+            # Override the MaximumRecordAgeInSeconds to make test shorter in LS
+            # which would otherwise fail validations (where MaximumRecordAgeInSeconds >= 60s).
```

Comment:
clarification: I didn't quite understand the second line of this comment. I guess it's to byepass AWS validations, which wouldn't allow setting it as low, right?

Idea: Maybe we can clearly label this as LS test optimization and say that we are patching the implementation directly to byepass AWS API validation 💭

Reply:
Would something along these lines be clearer and more specific?

> LocalStack test optimization: Override MaximumRecordAgeInSeconds directly in the poller to bypass the AWS API validation (where MaximumRecordAgeInSeconds >= 60s). This saves 55s waiting time.